lib/generators/acts_as_votable/migration/migration_generator.rb

```diff
@@ -24,7 +24,19 @@ def self.next_migration_number(path)
 
     def create_migration_file
       if self.class.orm_has_migration?
-        migration_template 'migration.rb', 'db/migrate/acts_as_votable_migration.rb'
+        migration_template 'migration.erb', 'db/migrate/acts_as_votable_migration.rb', migration_version: migration_version
+      end
+    end
+
+
+    private
+    def rails5?
+      Rails.version.start_with? '5'
```

Comment:
Could this be made more generic for future versions?

```
Rails.version.to_i >= 5
```

Reply:
rails 6 is live :)